inst/traduire/en-translation.json

```diff
@@ -276,5 +276,12 @@
     "DOWNLOAD_AGYW_DESCRIPTION": "Naomi AGYW tool uploaded from Naomi web app",
     "NUMBER_ON_ART": "Number on ART",
     "NUMBER_ON_ART_DESC": "Number on ART description",
-    "POPULATION_PROPORTION": "Population proportion"
+    "POPULATION_PROPORTION": "Population proportion",
+    "ANC_TOTAL_POS": "Anc total pos",
+    "ANC_TOTAL_POS_DESC": "Anc total pos desc",
+    "ANC_ALREADY_ART_DESC": "ANC already ART desc",
+    "ANC_STATUS": "ANC status",
+    "ANC_STATUS_DESC": "ANC status desc",
+    "ANC_ART_AMONG_KNOWN": "ANC ART among known",
+    "ANC_ART_AMONG_KNOWN_DESC": "ANC ART among known desc"
```

Comment:
"ANC total HIV positive"
"Total HIV positive women at ANC (known positive and tested positive)"
"Women already on lifelong ART before first ANC visit during this pregnancy"
"ANC HIV status ascertained"
"Number of women with HIV status ascertained during ANC (either known or tested at least once)"
"ANC ART coverage among known positive"  _this might be too long?_
"Proportion of women known HIV positive at first ANC visit who were already on ART"

Reply:
These look good to me